charts/server/values.yaml

```diff
@@ -30,81 +24,17 @@ common:
   # Ingress configuration
   # This overrides global.ingress, and can be overridden by component-specific settings
   ingress: {}
-  # Consul configuration
-  consul:
-    # The address of the Consul server
-    address: "{{ .Release.Name }}-consul-server"
-    # The port of the Consul server
-    port: 8500
-    # The tag to use for identifying Zenith services in Consul
-    serviceTag: "zenith-service"
-    # The prefix to use for additional service information for Zenith services in Consul KV
-    serviceKeyPrefix: "zenith/services"
-    # The prefix to use for registrar information in Consul KV
-    registrarKeyPrefix: "zenith"
+  # Common Kubernetes configuration
+  kubernetes:
+    # The target namespace for Zenith services
+    targetNamespace: zenith-services
```

Comment:
The problem with doing that is you need much more complicated and expensive logic to verify the uniqueness of the domains used across all namespaces, rather than just asking Kubernetes to do a create and getting a 409. This will also inevitably get racy.

Reply:
Personally, I'd prefer that these new CRDs are considered an internal Zenith implementation detail, like any other data store would be, and that any additional interactions with Zenith are implemented via the registrar API.

Reply:
FWIW, Dex does something similar when running on Kubernetes (i.e. uses CRDs as a store, but considers it an internal implementation detail)

Reply:
Ah, I see your point.